lisp/_prepare.el

```diff
@@ -1037,7 +1053,7 @@ Standard is, 0 (error), 1 (warning), 2 (info), 3 (log), 4 or above (debug)."
           (eask-warn "Unmatch dependency '%s'; add (depends-on \"%s\") to Eask-file or consider removing it" req req)))
       (dolist (dep dependencies)
         (unless (member dep requirements)
-          (eask-warn "Unmatch dependency '%s'; add (%s \"VERSION\") to package-file or consider removing it" dep dep))))))
+          (eask-warn "Unmatch dependency '%s'; add (%s \"VERSION\") to %s or consider removing it" dep dep def-point))))))
```

Comment:
Unmatch -> Unmatched.  Also above is a similar typo.

Reply:
And this! Thanks!